packages/data-core/src/store/store.md

```diff
@@ -1,9 +1 @@
 # store
```

Comment:
If this is all its gonna be, I'd just delete it. We'll set something up to auto-generate API docs. Don't need readmes in that case.

Reply:
sounds good, removed!